fuzzers/qemu_systemmode/build.rs

```diff
@@ -5,7 +5,7 @@ macro_rules! assert_unique_feature {
     () => {};
     ($first:tt $(,$rest:tt)*) => {
         $(
-            #[cfg(all(not(any(doc, feature = "clippy")), feature = $first, feature = $rest))]
+            #[cfg(all(not(doc), feature = $first, feature = $rest))]
```

Comment:
This change breaks clippy with `--all-features` I think

Reply:
Comparing with upstream, it breaks the same way. The `"clippy"` feature is not defined anywhere therefore ignored.
If I understand correctly the intended goal for this, the cfg `clippy` should do the job

Reply:
Should we add a clippy feature?

Reply:
I think so, but then we should do it in all build scripts.
I'd say it's another PR.

Reply:
Please don't! It should be a bare cfg, not a feature: https://blog.rust-lang.org/2024/02/28/Clippy-deprecating-feature-cargo-clippy.html

This must have been missed when I did the big transition for these a while ago. I agree this change should not be in this PR, but do not add a feature.

Reply:
yes indeed, we discussed this exact thing below :)
we need a separate pr to remove the feature in the crates that still use it so.

Reply:
i replaced this with clippy cfg but it looks like it doesn't work for build.rs ..?